autolab/Autolab
# Autofill scores for shared comments

## Description
* Upon selecting a shared comment, the score gets autofilled along with the comment.
* Auto-resizes the textarea when you click into it or select a shared comment

Resolves #1708, #1705

## Motivation and Context
Classes, such as 15-122, tend to pre-populate a rubric bank of responses, so that their TA's can easily select common responses. By filling in the scores along with the comments, there's one less step of TA's having to type in the scores themselves.

## How Has This Been Tested?
* Create shared annotation with a non-default score --> check that the shared annotation displays properly in the autocomplete dropdown and fills comment/score correctly
* Create shared annotation with a non-default score in global annotations --> check that the shared annotation displays properly in the autocomplete dropdown and fills comment/score correctly
* Create shared annotation with a non-default score --> check that the shared annotation displays properly in the autocomplete dropdown for global annotations and fills comment/score correctly
* Create shared annotation with a non-default score in global annotations --> check that the shared annotation displays properly in the autocomplete dropdown for global annotations and fills comment/score correctly
* Works across different submissions
* Works after editing score
* Works with shared comments created before this PR
* Autoresizes when long comment is selected
* Autoresizes when long comment is selected in global annotation

## Types of changes


## Checklist:


## Future Steps
* 15-122 also requested for the problem category to be auto-filled, but right now the shared comments show up per problem anyways. We could change this such that the shared comment pool is per assignment & then auto-fill problem, but we might need to discuss if this is a good idea.
* Could find some way to display score on autocomplete label
